lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -2270,6 +2352,23 @@ class EngineSemanticsOwner {
       return true;
     }());
   }
+
+  /// True, if any semantics node requested focus explicitly.
```

Comment:
Good point. I expanded the docs. The relevant time frame for this to be `true` is during a semantics update (in the middle of `updateSemantics` invocation). Otherwise, this field is almost always `false`. Simply put, if any node in the semantics update requests focus explicitly, then there's no need to apply default focus. So the flag is flipped to `true` to shut off the default. At the end of the update the flag is reset back to `false` allowing the default logic to be used on future updates.